_articles/challenge_administration.md

```diff
@@ -88,6 +88,8 @@ Throughout the challenge, participants will continuously submit to the evaluatio
 
 ### Revealing Submissions and Scores
 
-Organizers can create a leaderboard when scores are ready to be revealed to participants. Leaderboards are sorted, paginated, tabular forms that display submission annotations (e.g. scores from the scoring application and other metadata) and update as annotations or scores change. A leaderboard can provide real-time insight into the progress of a challenge.
+Organizers can create leaderboards when scores are ready to be revealed to participants using [Submission Views]({{ site.baseurl }}{% link _articles/views.md %#create-a-submission-view}).
```

Comment:
@marcomarasca Could you verify that the syntax on this tag is correct? I am attempting to build the site locally to test your edits/additions, and I am getting an error for this link. I am not a liquid expert (yet) so I can't spot the issue. Thanks!

Reply:
Thanks for spotting this @stacey-w-taylor, I think there is a typo. Will double check this!

Reply:
Fixed the anchor link!